paying_for_college/tests/test_views.py

```diff
@@ -116,12 +116,13 @@ def test_feedback(self):
         self.assertTrue(sorted(response.context_data.keys()) ==
                         ['base_template', 'form', 'url_root'])
 
-    @mock.patch('paying_for_college.views.render_to_response')
-    def test_feedback_post(self, mock_render):
-        client.post(
+    @mock.patch('paying_for_college.views.Feedback')
+    def test_feedback_post(self, mock_feedback):
+        response = client.post(
             reverse('disclosures:pfc-feedback'),
             data=self.feedback_post_data)
-        self.assertTrue(mock_render.call_count == 1)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(mock_feedback.call_count == 1)
```

Comment:
@willbarton I join you in not being entirely sure of the author's intent, but `test_feedback_creates_object` seems clearer in this context.